scrape/manager.go

```diff
@@ -238,11 +247,36 @@ func (m *Manager) ApplyConfig(cfg *config.Config) error {
 	}
 
 	c := make(map[string]*config.ScrapeConfig)
+	scrapeFailureLoggers := map[string]log.Logger{
+		"": nil,
```

Comment:
What do you gain from setting this explicitly?

Reply:
If you don't set it explicitly, `scrapeFailureLoggers[""]` will still return `nil`, right. That's what I was asking :) Just a nit anyway.